### ISSUE TITLE
teamd cannot turn off DEBUG messages

### ISSUE THREAD

**Issue by stcheng** [Azure/SONiC#48]
Jan 24 22:50:09 str-msn2700-04 DEBUG teamd_PortChannel32[402]: Ethernet36: lacp info state: 0x3D.
Jan 24 22:50:09 str-msn2700-04 DEBUG teamd_PortChannel40[405]: Ethernet40: lacp info state: 0x3D.
Jan 24 22:50:09 str-msn2700-04 DEBUG teamd_PortChannel48[408]: Ethernet52: lacp info state: 0x3D.
Jan 24 22:50:09 str-msn2700-04 DEBUG teamd_PortChannel40[405]: Ethernet44: lacp info state: 0x3D.
Jan 24 22:50:09 str-msn2700-04 DEBUG teamd_PortChannel16[396]: Ethernet16: lacp info state: 0x3D.

LACP info state is printed out every 1s even when the teamd debug level set to zero.

**Comment by lguohan** [comment]
```
	teamd_log_dbg("%s: lacp info state: 0x%02X.", lacp_port->tdport->ifname,
						      state);
```

change to teamd_log_dbgx should solve the problem.

**Comment by stcheng** [comment]
I don't think it is okay to change the dbg to dbgx, dbgx requires ctx as another argument.

**Comment by stcheng** [comment]
I would better delete this debug message for now and keep this issue open till we find the root cause.

**Comment by stcheng** [comment]
https://github.com/Azure/sonic-buildimage/pull/357
this is a temporary solution
